numpy/core/code_generators/cversions.txt

```diff
@@ -36,5 +36,6 @@
 0x0000000a = 9b8bce614655d3eb02acddcb508203cb
 
 # Version 11 (NumPy 1.13) Added PyArray_MapIterArrayCopyIfOverlap
-# Version 11 (NumPy 1.14) No Change
+# Version 11 (NumPy 1.14) Added PyArray_ResolveWritebackIfCopy,
+# PyArray_SetWritebackIfCopyBase and deprecate PyArray_SetUpdateIfCopyBase.
```

Comment:
Shouldn't this get a new hash on the last line? Or does that only happen at release time? @charris?

At any rate, I think the comment should go below the last hash in the file.

Reply:
It gets checked and updated at release time, but should be updated on the fly. IIRC, that was how it was done with the last addition in 1.13.